yarn.lock

```diff
@@ -17321,10 +17321,10 @@ react-native-webview@11.22.4:
     escape-string-regexp "2.0.0"
     invariant "2.2.4"
 
-react-native@0.69.3, react-native@0.69.4:
-  version "0.69.4"
-  resolved "https://registry.yarnpkg.com/react-native/-/react-native-0.69.4.tgz#d66f2a117442a9398b065876afdc568b209dc4da"
-  integrity sha512-rqNMialM/T4pHRKWqTIpOxA65B/9kUjtnepxwJqvsdCeMP9Q2YdSx4VASFR9RoEFYcPRU41yGf6EKrChNfns3g==
+react-native@0.69.3, react-native@0.69.5:
```

Comment:
Can you also update `apps/sandbox/package.json`, see https://github.com/expo/expo/pull/18743

Reply:
hm this app seems out of date, it still refers to expo@46.0.0-alpha.0